### DISCUSSION TITLE
rxjs 6 -> 7: Migration guide for TypeScript?

### DISCUSSION THREAD

**Discussion by tony** [ReactiveX/rxjs#6309] [Q&A]
Hi!

Thank you rxjs team for the excellent project!

Is anyone else experiencing hiccups with RXJS 6 -> 7?   If so, can anyone post what broke and what they did to fix it? (or what they weren't able to fix yet?)

Documentation of breaking changes in version 7: https://rxjs-dev.firebaseapp.com/deprecations/breaking-changes

**Comment by cartant** [comment]
FWIW, the docs site is still being tweaked. There is a more comprehensive list of changes here:

https://rxjs.dev/6-to-7-change-summary

Thanks for starting the discussion. If it highlights some TypeScript-related migration pain points, we can create a guide similar the deprecation-related refactoring guides.
